R/deprecate.R

```diff
@@ -362,6 +365,118 @@ loadFromHumann <- function(...) {
     importHUMAnN(...)
 }
 
+#' @rdname deprecate
+#' @export
+setGeneric("calculateDPCoA", signature = c("x", "y"),
+            function(x,y,...)
+                standardGeneric("calculateDPCoA"))
+
+#' @rdname deprecate
+#' @export
+setMethod("calculateDPCoA", c("ANY","ANY"), 
+            function(x,y,...) {
+                .Deprecated(msg = paste0("'calculateDPCoA' is deprecated. ",
+                                        "Use 'getDPCoA' instead."))
+                getDPCoA(x,y,...)
+            }
+)
+
+#' @rdname deprecate
+#' @export
+setMethod("calculateDPCoA", signature = c("TreeSummarizedExperiment","missing"),
+            function(x,y,...) {
+                .Deprecated(msg = paste0("'calculateDPCoA' is deprecated. ",
+                                        "Use 'getDPCoA' instead."))
+                getDPCoA(x,y,...)
+            }
+)
+
+#' @rdname deprecate
+#' @export
+setGeneric("calculateNMDS", function(x, ...) standardGeneric("calculateNMDS"))
```

Comment:
Yes. As scater still has the calculate* and run* naming scheme, we should still keep these as aliases.

You could do that by just adding the "regular" R function

```
calculateNMDS <- function(x, ...){
    getNMDS(x, ...)
}
```

Also do same for run* functions --> rename them to addRDA for instance and then create similarly alias runRDA

Reply:
Is it OK to add them as regular R function if I do not add the methods to the deprecate file? Here it deletes the methods names from the namespace file since they are not in the deprecate file but replaced with a regular function